pkgs/development/tools/rust/cargo-xbuild/default.nix

```diff
@@ -1,19 +1,19 @@
-{ stdenv, fetchFromGitHub, rustPlatform }:
+{ lib, fetchFromGitHub, rustPlatform }:
```

Comment:
Please don't make these unnecessary changes.

Reply:
Updated as your suggestion, thank you!